lib/src/default_transformer.dart

```diff
@@ -20,7 +20,14 @@ class DefaultTransformer extends Transformer {
   // TODO(nweiz): This should just take an AssetId when barback <0.13.0 support
   // is dropped.
   Future<bool> isPrimary(idOrAsset) {
-    var id = idOrAsset is AssetId ? idOrAsset : idOrAsset.id;
+    var id;
```

Comment:
I don't know what the issues are with dropping support for an old version. So, I think that's outside the scope of this work.